### PR TITLE
Add internationalization

### PR DESCRIPTION
**Pull request by RobertoCarrilloAvila**
# Description
I added the translations using `react-i18next` for all the components and I also added a prettier plugin in order to format the `.json` files used for the locales

### CODE REVIEW

File: src/components/Welcome/Welcome.js [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+        <li>{t('welcome.instructions')}</li>
         <li>
-          Create a target by clicking wherever on the map, specify the ratio and
-          and a topic: Travel, Dating, Music, etc.{' '}
-        </li>
-        <li>
-          <strong className="welcome--bold">target</strong> will start a chat
-          whenever you&lsquo;ve a match. You can always dismiss a conversation
-          if you&lsquo;re not interested.
+          <strong className="welcome--bold">{t('welcome.target')}</strong>{' '}
```
Comment (Ivan-Encinas) [inline src/components/Welcome/Welcome.js:25]: You don't need the empty brackets to make space

File: src/components/Welcome/Welcome.js [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
 const Welcome = () => {
+  const { t } = useTranslation();
   const { goTo } = useContentView();
 
   return (
     <div className="welcome">
       <img src={smiles} className="welcome__img" alt="smiles" />
       <h1 className="welcome__title">
-        Welcome to <strong className="welcome--bold">target</strong>
+        {t('welcome.welcome')}{' '}
```
Comment (Ivan-Encinas) [inline src/components/Welcome/Welcome.js:17]: Same here

File: src/components/Welcome/Welcome.js (hunk 1 above)
Comment (RobertoCarrilloAvila) [reply src/components/Welcome/Welcome.js]: Done!

File: src/components/Welcome/Welcome.js (hunk 2 above)
Comment (RobertoCarrilloAvila) [reply src/components/Welcome/Welcome.js]: Done!

**Pull request merged**